packages/insomnia-app/app/common/analytics.ts

```diff
@@ -114,7 +114,22 @@ export async function getDeviceId() {
 
 let segmentClient: Analytics | null = null;
 
-export async function trackSegmentEvent(event: string, properties?: Record<string, any>) {
+export enum SegmentEvent {
+  collectionCreate = 'Collection Created',
+  documentCreate = 'Document Created',
+  requestCreate = 'Request Created',
+  requestExecute = 'Request Executed',
+  testSuiteCreate = 'Test Suite Created',
+  testSuiteDelete = 'Test Suite Deleted',
+  unitTestCreate = 'Unit Test Created',
+  unitTestRunAll = 'Ran All Unit Tests',
+  unitTestRun = 'Ran Individual Unit Test',
+  unitTestDelete = 'Unit Test Deleted',
+  spaceLocalCreate = 'Local Space Created',
+  spaceLocalDelete = 'Local Space Deleted',
```

Comment:
```suggestion
  collectionCreate = 'Collection Created',
  documentCreate = 'Document Created',
  requestCreate = 'Request Created',
  requestExecute = 'Request Executed',
  spaceLocalCreate = 'Local Space Created',
  spaceLocalDelete = 'Local Space Deleted',
  testSuiteCreate = 'Test Suite Created',
  testSuiteDelete = 'Test Suite Deleted',
  unitTestCreate = 'Unit Test Created',
  unitTestDelete = 'Unit Test Deleted',
  unitTestRun = 'Ran Individual Unit Test',
  unitTestRunAll = 'Ran All Unit Tests',
```

any opposition to sorting these? :)

Reply:
Add a lint rule to automate it for me ;)

I don't like the wording either, but it means we can keep things grouped nicely as they get sorted. You win some, you lose some. I named them this way in anticipation of sorting haha.